student.rb

```diff
@@ -1,17 +1,16 @@
 class Student
+  include Teachable
+
   attr_reader :age, :phase
   attr_accessor :name
```

Comment:
I wrote how I'd do this in comments, but I don't understand why this is a good refactor. I'd rather this 3 lines of repetitive code in two files than to create another dependency. I'd love to better understand though.